src/storm/modelchecker/helper/conditional/ConditionalHelper.cpp

```diff
@@ -612,8 +621,15 @@ SolutionType computeViaBisection(Environment const& env, BisectionMethodBounds b
         if constexpr (storm::NumberTraits<SolutionType>::IsExact) {
             // find a rational number with a concise representation close to middle and within the bounds
             auto const exactMiddle = middle;
-            auto numDigits = storm::utility::convertNumber<uint64_t>(
-                storm::utility::floor(storm::utility::log10<SolutionType>(storm::utility::one<SolutionType>() / (*upperBound - *lowerBound))));
+
+            // Find number of digits - 1. Method using log10 does not work since that uses doubles internally.
+            auto numDigits = -1;
+            SolutionType remaining = storm::utility::one<SolutionType>() / (*upperBound - *lowerBound);
```

Comment:
Maybe put this into an aux function (in an anonymous namespace) called magnitude? Or even add magnitude to the constants.cpp?